Changelog.md

```diff
@@ -1,3 +1,9 @@
+### 0.5.15 (unreleased)
```

Comment:
I think we need to wait for the 0.6.0 release (when `develop_060` has been merged back to `develop`) and then change this line to `### 0.6.1 (unreleased)` after the rebase.